app/models/about.rb

```diff
@@ -82,7 +96,26 @@ def stats
       likes_last_day: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 1.days.ago).count,
       likes_7_days: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 7.days.ago).count,
       likes_30_days: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 30.days.ago).count
-    }
+    }.merge(plugin_stats)
+  end
+
+  def plugin_stats
+    final_plugin_stats = { plugin_stats: {} }
```

Comment:
Do we need to specify the key within the hash or should this be the job of the method caller? As far as this method is concerned, I think it should just return a hash of plugin stats.

Reply:
Good point, I will change the `merge(plugin_stats)` above to `merge(plugin_stats: plugin_stats)`